CHANGELOG.md

```diff
@@ -14,7 +14,10 @@ _None_
 
 ### Bug Fixes
 
-_None_
+* Workaround for Mint that does not yet support the SPM resource bundle.
```

Comment:
Two spaces after the `.`. It's unfortunate that the danger bot can't run on PR's from other repositories, or we'd have noticed this sooner.